cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -95,37 +90,37 @@ func main() {
 		os.Stdout.Write([]byte("\n"))
 		err := scanner.Err()
 		if err != nil {
-			klog.Fatal("Output monitoring failed! ", err)
+			fmt.Println("virt-v2v monitoring: Output monitoring failed! ", err)
```

Comment:
klog.Fatal also calls OsExit which we lose with this change

Reply:
added os.Exit in addition.